scripts/test_and_cover.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
-bin/wait-for --timeout=300 mariadb:3306
+bin/wait-for --timeout=300 mariadb:3306 mariadb_ht:3306
```

Comment:
```bash
bin/wait-for --timeout=300 mariadb:3306
# or set lower timeout for the second
bin/wait-for --timeout=300 mariadb_ht:3306
```

Reply:
Using upstream `wait-for` with a few additional tweaks seems to be working.